torch/_inductor/cudagraph_utils.py

```diff
@@ -90,7 +90,13 @@ def check_for_mutation(
 
     else:
         has_mutation = len(func.mutated_input_idxs) != 0
-        return None if not has_mutation else default_msg
+        return (
+            None
+            if not has_mutation
+            else format_default_skip_message(
+                f"mutated inputs ({len(func.mutated_input_idxs)} instances)"
+            )
```

Comment:
https://github.com/pytorch/pytorch/blob/main/test/inductor/test_cudagraph_trees.py#L255-L257

```
            FileCheck().check(
                "skipping cudagraphs due to mutation on input. Found from"
            ).check("torch.logical_xor").run(captured_output[0])
```

Are we not printing mutation traceback here ? where does this even gets printed

Reply:
probably from the `if torch._inductor.config.triton.cudagraph_trees:` block on line 75